src/data/carousel.yml

```diff
@@ -1,5 +1,5 @@
-- link: https://form.jotform.com/240367174269159
-  image: /images/carousel/2024 CWTS DAY banner.jpg 
+- link: zh/academic/degrees-programs/master-of-theological-essentials
```

Comment:
Add a / before the link

/zh/academic/degrees-programs/master-of-theological-essentials